<launch>
    <!--<rosparam command="load" file="$(find calib)/config/default_transform.yaml" />-->
    <node pkg="calib" name="cam1_cam2_publisher" type="tf_publisher_node" respawn="true" output="screen" args="cam1 cam2 cam_center" />
    <node pkg="calib" name="preproc_cam1" type="preprocessing_node" respawn="true" output="screen" args="/cam1/qhd/points cam1" />
    <node pkg="calib" name="preproc_cam2" type="preprocessing_node" respawn="true" output="screen" args="/cam2/qhd/points cam2" />
    <node pkg="calib" name="merging_cam1_cam2" type="merging_node" respawn="true" output="screen" args="cam1 cam2 scene" />
    <node pkg="calib" name="preproc_scene" type="preprocessing_node" respawn="true" output="screen" args="/calib/clouds/scene scene" />
    <node pkg="calib" name="tf_filter_cam1_cam2" type="tf_node" respawn="true" output="screen" args="cam_center cam1 cam2" />
</launch>
